package.json

```diff
@@ -22,10 +20,10 @@
   "author": "snyk.io",
   "license": "Apache-2.0",
   "dependencies": {
-    "debug": "^3.1.0"
+    "debug": "^3.1.0",
+    "lodash": "^4.17.10"
```

Comment:
If the goal is to have lodash de-duped, then we should be as flexible as possible (but consider the security constraints: https://snyk.io/vuln/npm:lodash). So I'd go for `^4.17.5`

Reply:
And of course, we should do the same for the CLI and any relevant plugins